server/prisma/schema.prisma

```diff
@@ -224,6 +225,22 @@ model GroupRoleOrder {
   @@map("groupRoleOrders")
 }
 
+model GroupTag {
+  groupId Int
+  tag     GroupTagType
+  index   Int
+
+  // Relations
+  group Group @relation(fields: [groupId], references: [id], onDelete: Cascade, onUpdate: NoAction)
+
+  // Constraints
+  @@id([groupId, tag, index])
```

Comment:
actually the constraint should be `[groupId, type]`, since we can't have the same tag type twice for the same groupId.